searchcore/src/vespa/searchcore/proton/attribute/attribute_writer.cpp

```diff
@@ -208,11 +209,11 @@ applyCompactLidSpace(uint32_t wantedLidLimit, SerialNum serialNum, AttributeVect
          * later config changes removing the attribute then it might
          * be smaller than expected during transaction log replay.
          */
-        attr.commit();
+        attr.commit(true);
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour. The same comment applies for the change further down.

Reply:
Removed semantic change